openslides/mediafiles/migrations/0002_auto_20150906_1246.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
@boehlke I know that this lines are added by django. But it is only necessary for the compatibility with python 2. OpenSlides needs python 3, so this line can be deleted.